src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -363,5 +363,51 @@ public async Task TestCredentialsCopyInAsyncContext()
                 server.Stop();
             }
         }
+
+        [Fact]
+        public void TestSmtpDeliveryFormatSevenBit()
```

Comment:
We usually follow the three part test name format MethodOrClassName_Scenario_ExpectedBehavior. Existing tests in this file are old, thus some of them don't follow the pattern.

You can rename the test as: `SendMailAsync_DeliveryFormatSevenBit_SubjectEncoded`.